lib/wallet/widgets/dialogs/your_ax_dialog.dart

```diff
@@ -4,199 +4,296 @@ import 'package:ax_dapp/wallet/widgets/ax_information_widgets.dart';
 import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
 import 'package:url_launcher/url_launcher.dart';
+import 'package:flutter/foundation.dart';
 
 class YourAXDialog extends StatelessWidget {
-  const YourAXDialog({super.key,});
+  const YourAXDialog({
+    super.key,
+  });
 
   @override
   Widget build(BuildContext context) {
     final _height = MediaQuery.of(context).size.height;
     final _width = MediaQuery.of(context).size.width;
+    final isWebMobile = kIsWeb &&
+        (defaultTargetPlatform == TargetPlatform.iOS ||
+            defaultTargetPlatform == TargetPlatform.android);
+    final isWebDesktop = kIsWeb &&
+        (defaultTargetPlatform == TargetPlatform.windows ||
+            defaultTargetPlatform == TargetPlatform.macOS ||
+            defaultTargetPlatform == TargetPlatform.linux);
     var wid = 400.0;
     const edge = 40.0;
     if (_width < 405) wid = _width;
     var hgt = 500.0;
     if (_height < 505) hgt = _height;
-      return Dialog(
-    backgroundColor: Colors.transparent,
-    shape: RoundedRectangleBorder(
-      borderRadius: BorderRadius.circular(12),
-    ),
-    child: Container(
-      height: hgt,
-      width: wid,
-      decoration: boxDecoration(Colors.grey[900]!, 30, 0, Colors.black),
-      child: SingleChildScrollView(
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-          children: [
-            SizedBox(
-              height: 80,
-              width: wid - edge,
-              child: Row(
-                mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                children: [
-                  Text(
-                    'Your AX',
-                    style: textStyle(Colors.white, 20, isBold:false),
-                  ),
-                  IconButton(
-                    icon: const Icon(
-                      Icons.close,
-                      color: Colors.white,
-                      size: 30,
+    return Dialog(
+      backgroundColor: Colors.transparent,
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.circular(12),
+      ),
+      child: Container(
+        height: hgt,
+        width: wid,
+        decoration: boxDecoration(Colors.grey[900]!, 30, 0, Colors.black),
+        child: SingleChildScrollView(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            children: [
+              SizedBox(
+                height: 80,
+                width: wid - edge,
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    if (isWebMobile)
+                      Center(
+                        child: Container(
+                          padding: const EdgeInsets.only(left: 20),
```

Comment:
Why are we adding padding to only on the left? does not make sense. It would make the metrics look lopsided. This also looks weird on desktop as well. Remember, when making UI changes like this, you would have to make sure that the UI does not change on desktop or vice versa.